Exdi/exdigdbsrv/yaml/Continuous-Integration.yml

```diff
@@ -0,0 +1,25 @@
+
+name: $(TeamProject)_$(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+- master
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  buildConfiguration: 'Release'
+  major: 1
+  minor: 0

```

Comment:
Please update the minor version to 1, since the revision counter will get reset with this change.

Reply:
done.